OpenRA.Mods.Common/Scripting/Properties/AircraftProperties.cs

```diff
@@ -16,12 +16,13 @@
 
 namespace OpenRA.Mods.Common.Scripting
 {
+	[ExposedForAIScript]
```

Comment:
Erm.. I think we are back to this issue, I originally split this in classes, not method level.